src/test/java/com/kodilla/ecommercee/CartDaoTestSuite.java

```diff
@@ -70,7 +70,7 @@ public void saveTest() {
 
         //CleanUp
         cartDao.deleteById(cartId);
-        productDao.deleteById(productId);
+       // productDao.deleteById(productId);
```

Comment:
Test wykazał że jak usuwacie koszyk to wszystkie produkty też są usuwane z bazy. Propozycją naprawy jest zakomentowanie kodu w teście który to wykrył. Powstają 2 pytania:
1) czy to jest prawidłowe działanie jeśli user doda produkty do koszyka, po czym się rozmyśli to ma możliwość usunięcia tych produktów z bazy?
2) czy nie lepiej by było naprawić przyczynę tego błędu?

Reply:
Znaleźliśmy rozwiązanie na ten problem. Zmieniliśmy w niektóych relacjach cascadeTypa. ALL na REFRESH, teraz przy tych ustawieniach  jak usune koszyk to produkty zostają w bazie.